### ISSUE TITLE
boot manager message and system logs appear during boot

### ISSUE THREAD

**Issue by alkazar** [gamer-os/gamer-os#15]
The boot sequence does not look nice. Should display an image during boot like SteamOS.

**Comment by sharkwouter** [comment]
SteamOS uses a background image for grub and the plymouth package to show an image during boot and shutdown. Plymouth is also used for showing a progress bar during system updates. Beware, though, plymouth can be a pain to get working with Nvidia drivers.

**Comment by sharkwouter** [comment]
I did some testing and I haven't been able to get plymouth to work on gamerOS. When I tried it on my desktop, it worked instantly. I'm pretty sure this is because my system uses EFI, which means plymouth can use high resolutions without any setup.

I would suggest for gamerOS to mainly support (U)EFI, since it is easier to work with on modern hardware and it would allow use to use plymouth more easily.
